front-end/studio/src/app/pages/apis/{apiId}/editor/_components/dialogs/add-response.component.ts

```diff
@@ -18,7 +18,134 @@
 import {Component, EventEmitter, Output, QueryList, ViewChildren} from "@angular/core";
 import {ModalDirective} from "ngx-bootstrap";
 import {Oas20Operation, Oas30Operation} from "oai-ts-core";
+import {DropDownOption, DropDownOptionValue as Value, DIVIDER} from "../../../../../../components/common/drop-down.component";
 
+export class HttpCode {
+
+    private code: number;
+    private name: string;
+
+    public constructor(code: number, name: string) {
+        this.code = code;
+        this.name = name;
+    }
+
+    public getCode(): number {
+        return this.code;
+    }
+
+    public getName(): string {
+        return this.name;
+    }
+}
+
+export class HttpCodeUtil {
+
+    private static HTTP_CODE_LIST_COMMON: HttpCode[] = [
+        new HttpCode(200, "OK"),
+        new HttpCode(201, "Created"),
+        new HttpCode(204, "No Content"),
+        new HttpCode(400, "Bad Request"),
+        new HttpCode(401, "Unauthorized"),
+        new HttpCode(403, "Forbidden"),
+        new HttpCode(404, "Not Found")
+    ];
+
+    private static HTTP_CODE_LIST: HttpCode[] = [
+        new HttpCode(100, "Continue"),
+        new HttpCode(101, "Switching Protocols"),
+        new HttpCode(102, "Processing"),
+        new HttpCode(200, "OK"),
+        new HttpCode(201, "Created"),
+        new HttpCode(202, "Accepted"),
+        new HttpCode(203, "Non_Authoritative"),
+        new HttpCode(204, "No Content"),
+        new HttpCode(205, "Reset Content"),
+        new HttpCode(206, "Partial Content"),
+        new HttpCode(207, "Multi-Status"),
+        new HttpCode(208, "Already Reported"),
+        new HttpCode(226, "IM Used"),
+        new HttpCode(300, "Multiple Choices"),
+        new HttpCode(301, "Moved Permanently"),
+        new HttpCode(302, "Found"),
+        new HttpCode(303, "See Other"),
+        new HttpCode(304, "Not Modified"),
+        new HttpCode(305, "Use Proxy"),
+        new HttpCode(306, "Switch Proxy"),
+        new HttpCode(307, "Temporary Redirect"),
+        new HttpCode(308, "Permanent Redirect"),
+        new HttpCode(400, "Bad Request"),
+        new HttpCode(401, "Unauthorized"),
+        new HttpCode(402, "Payment Required"),
+        new HttpCode(403, "Forbidden"),
+        new HttpCode(404, "Not Found"),
+        new HttpCode(405, "Method Not Allowed"),
+        new HttpCode(406, "Not Acceptable"),
+        new HttpCode(407, "Proxy Authentication Required"),
+        new HttpCode(408, "Request Time-Out"),
+        new HttpCode(409, "Conflict"),
+        new HttpCode(410, "Gone"),
+        new HttpCode(411, "Length Required"),
+        new HttpCode(412, "Precondition Failed"),
+        new HttpCode(413, "Payload Too Large"),
+        new HttpCode(414, "URI Too Long"),
+        new HttpCode(415, "Unsupported Media Type"),
+        new HttpCode(416, "Range Not Satisfiable"),
+        new HttpCode(417, "Expectation Failed"),
+        new HttpCode(418, "I'm a teapot!"),
```

Comment:
This code should only be in the list if the current date is April 1st. :)  

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418

Reply:
Nevermind - I see that you addressed this in the TS file.  Nice. :)